coderedcms/views.py

```diff
@@ -39,49 +36,35 @@ def search(request):
         search_query = search_form.cleaned_data['s']
         search_model = search_form.cleaned_data['t']
 
-        # get all codered models
-        pagemodels = sorted(get_page_models(), key=lambda k: k.search_name)
-        # get filterable models
+        # get all page models
+        pagemodels = sorted(get_page_models(), key=lambda k: k.__name__)
```

Comment:
Should use `search_name` here. Might need to refactor this to be an attribute on the class that gracefully handles a few levels of fallback.

Reply:
This cannot be an attribute on every page class, as this gets every page model, including wagtail pages.